staging/src/k8s.io/apiserver/pkg/server/genericapiserver.go

```diff
@@ -246,6 +246,17 @@ func (s preparedGenericAPIServer) Run(stopCh <-chan struct{}) error {
 	}
 
 	<-stopCh
+
+	if s.GenericAPIServer.AuditBackend != nil {
+		// TODO: make timeout configurable
+		shutdownStopCh := make(chan struct{})
+		go func() {
+			defer close(shutdownStopCh)
+			<-time.After(time.Minute)
```

Comment:
Let's start with this and TODO, once we reach a point where we'll gather some feedback we can decide what to do with it further.